pegen/build.py

```diff
@@ -79,9 +79,9 @@ def build_generator(
     with open(output_file, "w") as file:
         gen: ParserGenerator
         if output_file.endswith(".c"):
-            gen = CParserGenerator(rules.rules, file)
+            gen = CParserGenerator(rules, file)
```

Comment:
Looks like this argument should be called `grammar`. (Also in the `def` line above and for `PythonParserGenerator` below).